ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -97,11 +92,11 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
-          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
-          marginTop={0}
-          marginBottom={2}
+        <Text
+          variant={TextVariant.bodySm}
+          className="cancel-speedup-popover__description"
+          // gets passed through to Box component
+          display={DISPLAY.FLEX}
```

Comment:
The `Text` component accepts all Box component props so what you could do here is remove the padding and margin styles associated with the `cancel-speedup-popover__description` class and replace it with
```suggestion
          display={DISPLAY.FLEX}
          marginBottom={4}
```

Reply:
got it, wasn't sure the styling props are actually working and are prefered way of styling as an old-school CSS-belongs-into-CSS guy. But happy to follow this codebase's standards

Reply:
Nice! Thank you. I like writing CSS too. The utility props help save the amount of css that gets written so it does have it's benefits.